core/src/avm2/globals/flash/media/sound.rs

```diff
@@ -19,8 +19,6 @@ pub fn init<'gc>(
     args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
```

Comment:
Should Sound also be moved to an instance allocator?

Reply:
I added a test - it looks like Sound is a 'normal' object (the initialization happens during the `super()` call, not during the allocator).